TestingHelpers/MockFileSystem.cs

```diff
@@ -92,22 +92,29 @@ private string GetPathWithCorrectDirectoryCapitalization(string fullPath)
             return fullPath;
         }
 
-
         public MockFileData GetFile(string path)
         {
             path = FixPath(path);
             return GetFileWithoutFixingPath(path);
         }
 
+        private void SetEntry(string path, MockFileData mockFile)
+        {
+            path = FixPath(path, true);
```

Comment:
I'm personally a fan of pure/functional methods where possible, so I think I'd rather see this be immutable.

Reply:
You mean overwriting the value of the local variable? I'm not a fan of that either.